### DISCUSSION TITLE
Dependency reasoning for packageRules

### DISCUSSION THREAD

**Discussion by NotMyFault** [renovatebot/renovate#16649] [Request Help] [answered]
Hey everyone,

I'm currently in the process of setting up renovate for a repository. When using dependabot, I used yaml comments in the config to add additional reasoning why specific dependency versions are banned.

Considering renovate uses json for its config, comments are not an option, but maybe there's a similar option within packageRules or other to add up more information why a dependency version is excluded?

Thanks in advance!

**Comment by rarkins** [comment, marked as answer]
You can add a "description" field to any package rules for this purpose

**Reply by NotMyFault** [reply]
Thanks for the quick reply, I assume this is only for the config itself and doesn't appear in the dependency dashboard?
All good if it doesn't nervertheless.

**Reply by rarkins** [reply]
It's only on the config. Displaying it dashboard would be impractical because one branch can have many dependencies and each dependency may match many package rules

**Reply by NotMyFault** [reply]
> It's only on the config.

No worries, that's fine for my needs. I ended up with https://github.com/jenkinsci/jenkins/pull/6883/files

Thanks a lot 👍🏻
